MM2RandoLib/RandoSettings.cs

```diff
@@ -223,6 +223,17 @@ public PlayerSprite SelectedPlayer
             set => this.SetProperty(ref this.mSelectedPlayer, value);
         }
 
+        public ChargingSpeed HitPointChargingSpeed { get; set; } = ChargingSpeed.Slow;
+
+        public ChargingSpeed WeaponEnergyChargingSpeed { get; set; } = ChargingSpeed.Slow;
+
+        public ChargingSpeed EnergyTankChargingSpeed { get; set; } = ChargingSpeed.Normal;
+
+        public ChargingSpeed RobotMasterEnergyChargingSpeed { get; set; } = ChargingSpeed.Normal;
+
+        public ChargingSpeed CastleBossEnergyChargingSpeed { get; set; } = ChargingSpeed.Normal;
+
+
```

Comment:
I think the defaults that we ship with should use the fastest setting. Do you agree?

Reply:
agreed.